src/benchmarks/micro/corefx/System.Linq/Perf.OrderBy.cs

```diff
@@ -0,0 +1,151 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Engines;
+using MicroBenchmarks;
+
+namespace System.Linq.Tests
+{
+    class FavourColourComparer
+        : IComparer<System.Drawing.Color>
+    {
+        public static IComparer<System.Drawing.Color> Instance => new FavourColourComparer();
```

Comment:
this should be a readonly field to make sure that the allocation of comparer is not included in the benchmark results

```suggestion
        public static readonly IComparer<System.Drawing.Color> Instance = new FavourColourComparer();
```